schemas/common.json

```diff
@@ -288,32 +288,39 @@
             "title": "vCenter VM Folder",
```

Comment:
Do we still need this parameter

Reply:
Yes, the one I added is a different "host and clusters" folder